src/components/z-status-tag/styles.css

```diff
@@ -0,0 +1,132 @@
+z-status-tag {
+  --z-icon-width: 14px;
+  --z-icon-height: 14px;
+
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+
+  height: 22px;
+  border-radius: var(--border-radius);
+  padding-right: 4px;
+  padding-left: 4px;
+  text-transform: uppercase;
+
+  display: inline-flex;
+  align-items: center;
+}
+
+z-status-tag.expandable > z-icon {
+  margin-right: 0;
+  transition: margin-right 1s;
+}
+
+z-status-tag:not(.expandable) > z-icon:not(:last-child),
+z-status-tag.expandable:hover > z-icon:not(:last-child) {
+  margin-right: var(--space-unit);
+}
+
+z-status-tag.expandable > span {
+  max-width: 0;
+  transition: max-width 1s;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+z-status-tag.expandable:hover > span {
+  max-width: 12rem;
+}
+
+/*Yellow status-tag styles*/
+z-status-tag.yellow.default {
+  background-color: var(--status-warning-yellow-lighter);
+  color: var(--status-warning-yellow-dark);
+  fill: var(--status-warning-yellow-dark);
+}
+
+z-status-tag.yellow.light {
+  background-color: transparent;
+  color: var(--status-warning-yellow-dark);
+  fill: var(--status-warning-yellow-dark);
+}
+
+z-status-tag.yellow.dark {
+  background-color: var(--status-warning-yellow-dark);
+  color: var(--text-white);
```

Comment:
text color in the dark variants is not white (except for the neutral/gray variant). for example the text over the warning-dark variant is `#fbf4e6` (`--yellow50` in the new variables, `--status-warning-yellow-lighter` in the legacy ones)

Reply:
risolto